### ISSUE TITLE
Lower AUC score of pre-trained weights compared to the README

### ISSUE THREAD

**Issue by BumjunJung9287** [jfhealthcare/Chexpert#28]
Hi, Thanks for sharing your codes.

I tried testing on dev.csv (validation dataset that have about 200 cases) with the pretrained model in "config/pre_train.pth"
and got the AUC score like bellow
Cardiomegaly : 0.75921332
Edema: 0.91922399
Consolidation: 0.92703151
Atelectasis: 0.89496753
Pleural_Effusion: 0.91831263

mean AUC: 0.88375

and the mean AUC is lower than the results reported in your readme file where the mean AUC was 0.91336..
Can you let me know what is the reason that the pretrained model has lower AUC score?
I ran the code on the validation set of CheXpert original dataset.
Maybe I ran the code with different dataset?

Thank you for reading!

**Comment by yww211** [comment]
Is the pretrained model `config/pre_train.pth` a weight file trained from the `config/example.json` in this repo? @deadpoppy

**Comment by deadpoppy** [comment]
same question #30
